FluentUI/build.gradle

```diff
@@ -2,104 +2,81 @@
-    implementation 'com.android.support:appcompat-v7:28.0.0'
-    implementation 'com.android.support:exifinterface:28.0.0'
-    implementation 'com.android.support:recyclerview-v7:28.0.0'
-    implementation 'com.android.support:cardview-v7:28.0.0'
-    implementation 'com.android.support:design:28.0.0'
+    implementation fileTree(dir: 'libs', include: ['*.jar'])
+    implementation project(':fluentui_core')
+    implementation project(':fluentui_listitem')
+    implementation project(':fluentui_drawer')
+    implementation project(':fluentui_tablayout')
+    implementation project(':fluentui_others')
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
-    implementation 'com.jakewharton.threetenabp:threetenabp:1.1.0'
-    implementation 'com.splitwise:tokenautocomplete:2.0.8'
-    implementation "com.microsoft.device:dualscreen-layout:1.0.0-alpha01"
+    implementation "com.android.support:appcompat-v7:$supportVersion"
+    implementation "com.android.support:design:$supportVersion"
+    testImplementation 'junit:junit:4.12'
     androidTestImplementation 'com.android.support.test:runner:1.0.2'
     androidTestImplementation 'com.android.support.test.espresso:espresso-core:3.0.2'
-    testImplementation 'junit:junit:4.12'
-    testImplementation 'org.robolectric:robolectric:4.0-alpha-3'
-    implementation 'com.android.support.constraint:constraint-layout:1.1.3'
 }
-
 task sourceJar(type: Jar) {
     from android.sourceSets.main.java.srcDirs
     classifier "sources"
 }
 
-publishing {
-    repositories {
-        maven {
-            url 'https://pkgs.dev.azure.com/microsoftdesign/fluentui-native/_packaging/fluentui-android/maven/v1'
-            credentials {
-                username = project.hasProperty("mavenUserName") ? "$mavenUserName" : ""
-                password = project.hasProperty("mavenPassword") ? "$mavenPassword" : ""
-            }
-        }
-    }
-    publications {
-        FluentUI(MavenPublication) {
-            groupId 'com.microsoft.fluentui'
-            artifactId "FluentUIAndroid"
-            version = android.defaultConfig.versionName
-            artifact(sourceJar)
-            artifact(bundleReleaseAar)
-            pom.withXml {
-                def dependenciesNode = asNode().appendNode('dependencies')
-                // Iterate over the compile dependencies (we don't want the test ones), adding a <dependency> node for each
-                configurations.implementation.allDependencies.each {
-                    if (it.group != null && (it.name != null && it.name != "unspecified") && it.version != null) {
-                        def dependencyNode = dependenciesNode.appendNode('dependency')
-                        dependencyNode.appendNode('groupId', it.group)
-                        dependencyNode.appendNode('artifactId', it.name)
-                        dependencyNode.appendNode('version', it.version)
+project.afterEvaluate {
```

Comment:
whats the purpose of adding afterEvaluate here

Reply:
Will be removed in next patch